frontend/packages/dev-console/src/components/import/import-validation-utils.ts

```diff
@@ -39,6 +39,42 @@ export const validationSchema = yup.object().shape({
         message: 'This field cannot be empty.',
       }),
   }),
+  serverless: yup.object().shape({
+    enabled: yup.boolean(),
+    scaling: yup.object().when('enabled', {
+      is: true,
+      then: yup.object({
+        minpods: yup
+          .number()
+          .integer('Min Pods must be an Integer.')
+          .min(0, 'Min Pods must be greater than or equal to 0.')
+          .test({
+            name: 'isEmpty',
+            test: (value) => value !== undefined,
```

Comment:
The help text says that if unspecified, defaults to zero, but this rule will not let the user remove the value?